concepts/05 UI Components/DataGrid/20 Editing/45 Disable Editing.md

```diff
@@ -0,0 +1,46 @@
+DataGrid has API to disable editing a specific column, row and cell.
```

Comment:
I don't like this sentence, the verb "has" is very weak. I suggest to rewrite the intro. Add more details about what a customer can find in this topic.

Reply:
ok